### PR TITLE
Zijie lint fix for reports component Phase 7

### PR DESCRIPTION
**Pull request by iTvX**
# Description
Lint fix for Reports component, this is phase 7.

## Files changed
All js/jsx file in /src/components/Reports/TasksDetail

eslint test:
CleanShot 2023-09-30 at 18 38 58@2x

# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Check all the functionality of the report is the same as before, and see if any bugs have appeared. (Please record a video)

(Optional)
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- Delete .eslintignore file in the root: `rm .eslintignore`
- `npx eslint src/components/Reports/TasksDetail/TasksDetail.jsx`
- `npx eslint fix src/components/Reports/TasksDetail/index.js`
- `git restore .`

I have complete confidence that **the code will pass the ESLint tests.** Therefore, I kindly request that you **focus on testing the frontend**. Specifically, please ensure that the **report functionality remains consistent** with how it operated previously. Thank you for your time and effort!

### CODE REVIEW

**Comment by beblicarl** [comment]
There were no errors and the Reports component works as before

[screencast-localhost_3000-2023.10.01-14_37_11.webm](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/68056104/04fe941d-9a1e-410a-bd1b-3e811b2e1239)

**Review by beblicarl — APPROVED** [review]

**Comment by luisarevalo21** [comment]
Hello @iTvX 
I tested your PR, the app runs as normal, however, when running the optional tasks, errors occur, should this be happening? 
This happened in a previous PR as well. 
Screenshot 2023-10-02 at 8 03 02 PM

**Comment by iTvX** [comment]
> Hello @iTvX I tested your PR, the app runs as normal, however, when running the optional tasks, errors occur, should this be happening? This happened in a previous PR as well. Screenshot 2023-10-02 at 8 03 02 PM

It seems like you are not in the root of the file, so you can not find what you want.

**Comment by DavidC0126** [comment]
Hi @iTvX I tested your Pr and it works as intended, however, sometimes those three buttons to show the charts would cause too much loading time (I cannot get the charts sometimes).


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/74082492/ec4ef389-f88e-4dd2-a9b0-a085b7b1b2f9

**Review by luisarevalo21 — APPROVED** [review]
Hello @iTvX,

I rested your PR and works as intended. Great work! 
Running the eslint scrip shows no issues. Although, I had to add --fix to your script. 
Lastly, the application runs as normal. 


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/f5cfa41d-69c1-4f26-813c-e1497c7fd4ca

Screenshot 2023-10-05 at 8 25 24 PM

**Review by Jiyuanxia — DISMISSED** [review]
Hi @iTvX , I have reviewed your pull request and tested it. It works as same as dev branch. Well done!

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92963742/f5e18d46-e13b-4426-92d3-8fa1899a5735

**Review by Shubhu0 — APPROVED** [review]
Hi @iTvX Tested your PR and it working as intended!

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/56580102/b2bb029b-b969-44e9-8de8-bcbcb75ff374

**Review by savio-henrique — DISMISSED** [review]
Hi @iTvX , I tested the PR and the app is working as intended. ESLint doesn't show any errors also.
terminal

**Review by yi-lin-1234 — APPROVED** [review]
Hello, I tested your PR and works as intended. Great work!


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92185695/89e08596-3516-4022-b13b-b4b64284b4f2


Screenshot 2023-10-25 at 10 17 54 AM

**Review by Chuehleo — APPROVED** [review]
Everything works as expected, great job!

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/122568562/48383fa0-3654-4450-83a9-f3991f8d4199

Screenshot 2023-10-26 000909

File: src/components/Reports/TasksDetail/TasksDetail.jsx [hunk 1]
```diff
@@ -1,51 +1,49 @@
+// eslint-disable-next-line no-unused-vars
```
Comment (EvianTan) [inline src/components/Reports/TasksDetail/TasksDetail.jsx:1]: Same comment as [here](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1356/files#r1384348686)

File: src/components/Reports/TasksDetail/index.js [hunk 2]
```diff
@@ -1 +1,2 @@
-export { TasksDetail } from './TasksDetail';
+// eslint-disable-next-line import/prefer-default-export
```
Comment (EvianTan) [inline src/components/Reports/TasksDetail/index.js:1]: Please fix it instead of disabling it.

Comment (iTvX) [reply src/components/Reports/TasksDetail/index.js]: fixed

File: src/components/Reports/TasksDetail/TasksDetail.jsx (hunk 1 above)
Comment (iTvX) [reply src/components/Reports/TasksDetail/TasksDetail.jsx]: fixed

**Comment by EvianTan** [comment]
image
Failed to build and cannot login via incognito window, Same as this [comment](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1356/files#r1391983106)

**Pull request closed without merging**